src/BugReporting.jl

```diff
@@ -29,19 +44,17 @@ end
 
 function rr_pack(trace_directory)
     check_rr_available()
-
-    trace_directory = normalize_inner_trace(trace_directory)
+    
     rr() do rr_path
-        run(`$rr_path pack $(trace_directory)`)
+        for dir in collect_inner_traces(trace_directory)
+            if !any(startswith(f, "mmap_pack_") for f in readdir(dir))
```

Comment:
I'm not sure we should do this. Sometimes if rr pack gets interrupted it'll leave these files around while not being actually packed.

Reply:
Okay.  I'll just have it unconditionally pack every time.